### DISCUSSION TITLE
Is it possible to add multiple bookmarks at once?

### DISCUSSION THREAD

**Discussion by siainbuletin** [xwmx/nb#346] [Q&A] [answered]
Often I find a bunch of articles that I want to quickly dump in the bookmarks notebook that I made with `nb`. But I can only add them one by one. I don't have a lot of knowledge of how I could manipulate this prompt, so the only thing I tried and saw wasn't working was `nb <url1> <url2>`. 

Is there any way to do this, or would a new command have to be defined?

**Comment by xwmx** [comment, marked as answer]
@siainbuletin This is now available as of version 7.15.0. Multiple URLs can be bookmarked with a single command by passing multiple `<url>` arguments. Additional arguments will be reused for each bookmark:

```bash
❯ nb https://example.com https://example.net --tags tag1,tag2 --filename example
Added: [1] 🔖 example.bookmark.md "Example Domain (example.com)"
Added: [2] 🔖 example-1.bookmark.md "Example Domain (example.net)"
```

**Reply by kotp** [reply]
You could also use your shell to do so with a for loop.

It is nice that it is available via the tool now, as well.
